context/context.go

```diff
@@ -78,6 +79,8 @@ type Context interface {
 	StmtRollback()
 	// StmtGetMutation gets the binlog mutation for current statement.
 	StmtGetMutation(int64) *binlog.TableMutation
+	// StmtDirtyTablesOP logs the dirty tables operations for current statement.
+	StmtLogDirtyOP(op int, tid int64, handle int64, row []types.Datum)
```

Comment:
`Log` can be misunderstood as write log message to file.
How about `StmtAddDirtyTableOp`?

Reply:
Done. @coocood